### PR TITLE
add graphql-voyager to console, close #314

### LINKED ISSUE #314: View in voyager [proposal]
What about something like a button “View in voyager” that opens up the Schema in an embedded voyager screen!

[link to graphql-voyager!](https://github.com/APIs-guru/graphql-voyager)

### PR DESCRIPTION
**Pull request by arvi3411301**
### Description

Todo:
- [x] Add Voyager Btn next to Explorer
Screenshot from 2019-06-06 17-43-14

- [x] Move voyager view inside hasura console container (like graphiql)

### Affected components 

- Console

### Related Issues

Fix #314 , #1731

### CODE REVIEW

File: console/src/components/Main/Main.js [hunk 1]
```diff
@@ -614,7 +614,10 @@ class Main extends React.Component {
             </div>
           </div>
 
-          <div className={styles.main + ' container-fluid'}>
+          <div
+            id="right_container_wrapper"
```
Comment (rikinsk) [inline console/src/components/Main/Main.js]: why is this needed? If needed, rename to main_wrapper. It is not wraapping right_container

**Comment by corysimmons** [comment]
@arvi3411301 @rikinsk Any idea why this isn't in the Hasura Console anymore?

Seemed like a really cool feature.

image
